app/experimenter/experiments/constants/nimbus.py

```diff
@@ -186,19 +186,6 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.FENIX.name, Application.IOS.name),
 )
 
-TARGETING_HOMEPAGE_GOOGLE = NimbusTargetingConfig(
-    name="Homepage set to google.com",
-    slug="homepage_google_dot_com",
-    description="Users with their Homepage set to google.com",
-    targeting=(
-        "!homePageSettings.isDefault && "
-        "homePageSettings.isCustomUrl && "
-        "homePageSettings.urls[.host == 'google.com']|length > 0"
```

Comment:
@ncloudioj `homePageSettings` evaluates to `undefined` in Firefox 93 so it will have to be changed or removed for our integration tests to pass.